src/datajudge/db_access.py

```diff
@@ -9,6 +9,7 @@
 from typing import Callable, Sequence, final, overload
 
 import sqlalchemy as sa
+from snowflake.connector.cursor import SnowflakeCursor
```

Comment:
Guard this import
```python
try:
    from...
except ImportError:
    print("a useful message")
```

Reply:
Btw nowadays we have `ModuleNotFoundError`

Reply:
I removed the import statement since it only is a type hint for the user

Reply:
Then you should do
```
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from ....
```

Reply:
I don't think I really get what you mean.

I initially added the import to have a 
```py
cursor: SnowflakeCursor = engine.connect().connection.cursor()
```
line, where the IDE then _knows_ that cursor has additional methods provided by snowflake.

Since this was not needed anymore (not important enough) I removed it again, so it had no checking functionality. So it should be fine to just remove it.